config/settings.yml

```diff
@@ -35,3 +35,9 @@ BORROW_DIRECT_CODE: BORROW_DIRECT
 
 schedule_once:
   enabled: false
```

Comment:
I've kept this top-level flag as a convenience to not have to check both accessors before rendering a wrapping element.  Would be happy to 🔪 this though.